test/test_atomix_metal.jl

```diff
@@ -69,6 +69,21 @@ end
     @test collect(A) == [2, 1, 2]
 end
 
+@testset "AtomixMetalExt:test_inc_threadgroup" begin
+    A = Metal.MtlVector(Float32(1):Float32(3))
+    metal() do
+        GC.@preserve A begin
+            B = Metal.MtlThreadGroupArray(Float32, 3)
```

Comment:
You forgot to initialize the thread-local memory:

```suggestion
            B = Metal.MtlThreadGroupArray(Float32, 3)
            B[1] = A[1]
```